src/main/java/seedu/address/model/ModelPredicate.java

```diff
@@ -22,4 +23,15 @@
         person -> person.getInfectionStatus().getStatusAsBoolean();
     public static final Predicate<Person> PREDICATE_SHOW_ALL_QUARANTINED =
         person -> person.getQuarantineStatus().getStatusAsBoolean();
+
+    /** {@code Predicate} to handle location list */
```

Comment:
The JavaDocs for this Predicate could be clearer as this Predicate is not as straightforward as the others.